components/eamxx/docs/_presentations_srcs/eamxx-tutorial-2025-part2.md

```diff
@@ -487,7 +487,7 @@ void POMPEI::run_impl(const double dt) {
   using PC = scream::physics::Constants<Real>;
   auto g   = PC::gravit;
 
-  auto t    = timestamp() + dt;
```

Comment:
@mahf708 Even though _at the time of the tutorial_ `timestamp()` was correct, I decided to change this, for the sake of who comes later and reads the docs.